trade-finance-manager-api/src/v1/controllers/amendment.controller.js

```diff
@@ -210,15 +236,17 @@ const updateFacilityAmendment = async (req, res) => {
       }
 
       // UKEF exposure
-      payload = calculateUkefExposure(payload);
+      payload = calculateACBSUkefExposure(payload);
```

Comment:
Can we please follow camel case, please rename it to `caculateAcbsUkefExposure`. Thanks

Reply:
done